book/website/community-handbook/community-calls/community-calls-steering-committee.md

```diff
@@ -0,0 +1,15 @@
+(ch-calls-steering-committee)=
+# Steering Committee Meetings 
+
+```{warning} This page is under review and will be updated with more details soon.
+```
+## Steering Committee Meetings
+In 2025, Steering Committee Meetings will be hosted regularly to formalise Constitution Level governance.
+These meetings are attended by drawing from chairs and representatives of Working Groups and Delivery Groups (projects carried out in collaboration with _The Turing Way_).
```

Comment:
A reference for working and delivery groups would be good.
It looks like the page in the forward is missing labels so we would have to add one:eyes:.